aqt/updater.py

```diff
@@ -85,14 +85,24 @@ def _detect_qmake(self) -> bool:
                 return True
         return False
 
-    def patch_pkgconfig(self):
+    def patch_pkgconfig(self, os_name):
         for pcfile in self.prefix.joinpath("lib", "pkgconfig").glob("*.pc"):
-            self.logger.info("Patching {}".format(pcfile))
-            self._patch_textfile(
-                pcfile,
-                "prefix=/home/qt/work/install",
-                "prefix={}".format(str(self.prefix)),
-            )
+            if os_name == "linux":
+                self.logger.info("Patching {}".format(pcfile))
+                self._patch_textfile(
+                    pcfile,
+                    "prefix=/home/qt/work/install",
+                    "prefix={}".format(str(self.prefix)),
+                )
+            elif os_name == "mac":
+                self.logger.info("Patching {}".format(pcfile))
+                self._patch_textfile(
+                    pcfile,
+                    "prefix=/Users/qt/work/install",
```

Comment:
There is only a difference is a value here.
I prefer to determine os_name  in update() function, and pass the value `oldprefix='/Users/qt/work/install'` as an argument of patch_pkgconfig function. It will reduce a code duplication.

Reply:
done.